core/app/models/spree/inventory_unit.rb

```diff
@@ -118,6 +118,10 @@ def allow_ship?
       on_hand?
     end
 
+    def value_attributes
+      attributes.slice('pending', 'state', 'variant_id', 'line_item_id')
```

Comment:
Probably could use the symbols here.

Reply:
attributes is not a hash with indifferent access :(